zerver/views/__init__.py

```diff
@@ -233,7 +238,10 @@ def accounts_register(request):
 
         if first_in_realm:
             do_change_is_admin(user_profile, True)
-            return HttpResponseRedirect(reverse('zerver.views.initial_invite_page'))
+            invite_url = reverse('zerver.views.initial_invite_page')
+            if (realm_creation and settings.REALMS_HAVE_SUBDOMAINS):
+                invite_url = settings.EXTERNAL_URI_SCHEME + form.cleaned_data['realm_subdomain'] + "." + settings.EXTERNAL_HOST + reverse('zerver.views.initial_invite_page')
```

Comment:
This line is super long.  Maybe just 
```
"%s%s.%s%s" % (
    a,
    b,
    c,
    d,
}
```
where a,b,c,d are the 4 big components of the URL